verify_docs.py

```diff
@@ -6,22 +8,62 @@
 
 import sys
 import yaml
+import copy
+
+from distutils.version import StrictVersion
+
+def verify_swagger(spec):
+    errored = False
+    for i in spec["paths"]:
+        for j in spec["paths"][i]:
+            k = spec["paths"][i][j]
+            if "summary" not in k:
+                print("Error: summary missing from Swagger doc in: %s " % (i))
+                errored = True
+    return errored
+
+def verify_openapi(spec):
+    errored = False
+    for i in spec["paths"]:
+        for j in spec["paths"][i]:
+            k = spec["paths"][i][j]
+            if "summary" not in k:
+                print("Error: summary missing from OpenAPI doc in: %s " % (i))
+                errored = True
+    return errored
```

Comment:
These are the exact same logic, right? Except for the error message...

Not strong opinion on leaving it as-is if in the future we might have different checks on them. tt just felt so weird (I did check it line by line...).

Reply:
I was thinking of adding logic for checking unused references, but it turned out to be a little more complicated than I initially thought.